src/main/java/io/hops/transaction/handler/TransactionalRequestHandler.java

```diff
@@ -151,14 +151,14 @@ protected Object execute(Object info) throws IOException {
           for(String subOpName: subOpNames){
             opName = opName + ":" + subOpName;
           }
+          String inodeLocks = locksAcquirer != null ? getINodeLockInfo(locksAcquirer.getLocks()):"";
           requestHandlerLOG.warn(opName + "TX Failed. " +
                   "TX Time: " + (System.currentTimeMillis() - txStartTime) + " ms, " +
                   // -1 because 'tryCount` also counts the initial attempt which is technically not a retry
                   "RetryCount: " + (tryCount-1) + ", " +
                   "TX Stats -- Setup: " + setupTime + "ms, AcquireLocks: " + acquireLockTime + "ms, " +
                   "InMemoryProcessing: " + inMemoryProcessingTime + "ms, " +
-                  "CommitTime: " + commitTime + "ms. Locks: "+
-                  getINodeLockInfo(locksAcquirer.getLocks())+". " + t, t);
+                  "CommitTime: " + commitTime + "ms. Locks: "+inodeLocks+". " + t, t);
```

Comment:
There is the same problem line 139 when trace log is enabled, can you fix it too?

Reply:
If it gets to line 139 then It mean that every thing went ok and lock acquirer was initialized on line 86.

Reply:
Yea, sorry.